lib/ansible/modules/cloud/amazon/cloudformation.py

```diff
@@ -346,6 +346,21 @@ def create_changeset(module, stack_params, cfn):
             result = dict(changed=False, output='ChangeSet %s already exists.' % changeset_name, warnings=[warning])
         else:
             cs = cfn.create_change_set(**stack_params)
+            while True:
```

Comment:
We can't leave this 100% unbounded, in general a higher timeout (10 minutes should be sufficient for most every stack changesets) is fine, and avoids us accidentally creating an infinite loop.

Reply:
Good point, has been fixed.